examples/undocumented/python/distance_canberraword.py

```diff
@@ -10,20 +10,16 @@
 parameter_list = [[traindna,testdna,3,0,False],[traindna,testdna,3,0,False]]
 
 def distance_canberraword (fm_train_dna=traindna,fm_test_dna=testdna,order=3,gap=0,reverse=False):
-	from shogun import StringCharFeatures, StringWordFeatures, DNA
-
-	charfeat=StringCharFeatures(DNA)
-	charfeat.set_features(fm_train_dna)
-	feats_train=StringWordFeatures(charfeat.get_alphabet())
-	feats_train.obtain_from_char(charfeat, order-1, order, gap, reverse)
+	charfeat=sg.create_string_features(fm_train_dna, sg.DNA)
```

Comment:
sg.dna -> "DNA" ?

Reply:
That might be tricky because this actually uses the constructor rather than put/get

Reply:
It needs to be though as the type won’t be available in the future...
But can be postponed I guess

Reply:
Can have a dispatch in StringFeatures for that

Reply:
couldnt we just force the user to set this with put rather than in the factory?